src/braket/simulator/braket_simulator.py

```diff
@@ -63,3 +63,33 @@ def run(
     @abstractmethod
     def properties(self) -> DeviceCapabilities:
         """DeviceCapabilities: Properties of the device."""
+
+
+class MultiSimulator(BraketSimulator):
```

Comment:
It seems like a little overkill to create a new abstract class/interface this one method that's very related to the existing run method.

Reply:
Yes, I'm also curious why we couldn't just add `run_multiple` to the existing interface, and provide a default implementation.

Reply:
Regardless, we're going to need a new method for the backend to run multiple circuits; this means it has to live either in BraketSimulator or a new class. If it lives in `BraketSimulator`, then the SDK would need either check whether the method throws an exception (not great), or we'd need to add a new property in `BraketSimulator` for the SDK to check, which then leads to a situation where devs can implement the method and forget to mark the property true, or vice-versa.

Reply:
Could `BraketSimulator` also provide a default implementation of `run_multiple`? (maybe it could do exactly what the SDK does today for `run_batch` - delegate to `Pool`)

Reply:
I suppose it could; it just doesn't seem useful to have the same implementation in both the simulator _and_ the SDK; as well, the abstract `BraketSimulator` class having a default implementation could potentially lead to multiple inheritance issues down the line.

Reply:
Alternative implementation in #264